src/ImageSharp/Memory/BufferArea{T}.cs

```diff
@@ -120,7 +123,7 @@ public BufferArea<T> GetSubArea(int x, int y, int width, int height)
         public BufferArea<T> GetSubArea(Rectangle rectangle)
         {
             ImageSharp.DebugGuard.MustBeLessThanOrEqualTo(rectangle.Width, this.Rectangle.Width, nameof(rectangle));
-            DebugGuard.MustBeLessThanOrEqualTo(rectangle.Height, this.Rectangle.Height, nameof(rectangle));
+            SixLabors.DebugGuard.MustBeLessThanOrEqualTo(rectangle.Height, this.Rectangle.Height, nameof(rectangle));
 
             int x = this.Rectangle.X + rectangle.X;
```

Comment:
We're mixing two `DebugGuard` methods now. We should ditch the ImageSharp one.

Reply:
Good catch! Having common internals is just too dangerous, we should probably drop `[InternalsVisibleTo(...)]` in SL.Core to avoid these issues.

Reply:
Yeah, though I was wondering whether we should just make them public and avoid duplication? They're useful and well tested.

Reply:
One can accidentally include and use such internals with R# (been there). I would rather share them through submodules in the future. For now it's just safer to hide them IMO.

Reply:
Hide them then. We can investigate private packages like they use for ASP Core in the future